Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,16 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    if official?
+      "https://github.com/Homebrew/homebrew-#{repo.strip_prefix("homebrew-")}/issues"
+    else
+      "https://github.com/#{name}/issues"
```

Comment:
`repo` variable doesn't include `homebrew-` part. So you could use `https://github.com/#{user}/homebrew-#{repo}/issues` directly.

In addition, what shall we show for custom git remote? 

So in all, we may want something like:
```ruby
def issues_url
  if custom_remote?
    remote # what to show, just remote?
  else
    "https://github.com/#{user}/homebrew-#{repo}/issues"
  end
end